### PR TITLE
[T4A6][W15-B4]Jasper Lee

### PR DESCRIPTION
**Pull request by jasperlee27**
Ready for review

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:3]: don't import everything. import only what is necessary.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+public class UtilsTest {
+
+    @Test
+    public void isNoAnyNull() {
+        assertFalse(Utils.isAnyNull("qwerty", 98765431, "guy@girl.com"));
+    }
+
+    @Test
+    public void isYesAnyNull() {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:17]: the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        testObjects.add(new Integer(987654321));
+        testObjects.add(new String("guy@girl.com"));
+        testObjects.add(new String("qwerty"));
+        assertFalse(Utils.elementsAreUnique(testObjects));
+    }
+
+    @Test
+    public void elementsAreYesUnique() {
+        ArrayList<Object> testObjects = new ArrayList<Object>();
+        testObjects.add(new String("qwerty"));
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:34]: going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.

**Comment by okkhoy** [comment]
@jasperlee27 
Some comments added. Please ack & close the PR after reading comments.

**Comment by jasperlee27** [comment]
Acknowledged. Thankyou!

**Pull request closed without merging**